docs/meshstack.logging.md

```diff
@@ -0,0 +1,177 @@
+---
+id: meshstack.logging
+title: Logging & Auditing
+---
+
+The meshStack system consists of different components. On the one hand there are meshStack components, like the backend, replicators and further microservices. Additionally, Keycloak is used as an authentication and authorization system. Logs are therefore distributed among multiple components. In order to provide central access to the log files for administrators, a central logging system like an ELK (Elasticsearch, Logstash, Kibana) stack is used. It allows searching logs of multiple components.
```

Comment:
You could add a link the the Keycloak website if someone wants to get more info about it